### ISSUE TITLE
Custom study menu should have Increase Today's new/review card limit

### ISSUE THREAD

**Issue by Arthur-Milchior** [ankidroid/Anki-Android#17666]
How to reproduce:
* Create a deck Parent::Child
* ensure there is no cards in Parent
* have new cards in Child
* long tap Parent
* tap custom study

Expected result:
* as in anki, there should be the options to increase today's new or review card limit

![Image](https://github.com/user-attachments/assets/7e3ac107-0185-48b3-bb07-35b8f6e571cb)

Actual result:
* this option is not present

**Comment by david-allison** [comment]
Why? The dialog will close with a no-op nothing if there's no new cards available

**Comment by Arthur-Milchior** [comment]
Even if there are new cards we could show, the option to increase the number of new cards is not presented in ankidroid's custom study.
I understand I selected an absurd screenshot, with it's zero new cards. Sorry for that. Is it what caused your remark regarding the no-op?

**Comment by david-allison** [comment]
Yes, because we support the functionality but only display it when it's available

**Comment by Arthur-Milchior** [comment]
After more investigation, I just understood what's wrong here. It's broken when the new cards are present in subdeck only.
I didn't realize this was important and so omitted it from my reproduction steps.
I edited it.
Would you agree, now, that there is an issue to solve?

**Comment by Arthur-Milchior** [comment]
Okay, my bad. I realize that, at least on anki, the feature is enabled, it does not throw an error. However, it does not actually lead to more reviews. The cards in subdecks are ignored. I'll go ask anki whether that's the expected behavior. The manual is not clear about it, and, worse, the number of cards in subdeck is shown, so I'd assume it actually matters.

**Comment by Arthur-Milchior** [comment]
Also, but this a details I find less impoRtant. I prefer when actions that are unavailable are disabled rather than removed. Or even enable the radio button and disable the "ok" button.

Here is the scenario that confused me:
I was done with my current deck. I wanted to add more cards. So I used custom study. And here I thought "that's strange, why do I not see the option to add more cards. I was sure it was there. I must have mixed it with another menu. Or maybe it was moved recently". I would have appreciated something telling me that there is no more new cards and that I'll only have reviews from now on.

**Comment by Arthur-Milchior** [comment]
I won't confirma nything, because I don't even know if the current behavior is the expected one or a bug. So I'll wait for DAE's feedback

**Comment by david-allison** [comment]
I wasn't mistaken in the logic, this matches Anki Desktop: Only the current deck is taken into account

BUT: I'll get in a fix to disable the icons, rather than removing them

**Comment by Arthur-Milchior** [comment]
After trying to show disabled entires in the list, I realized that the AlertDialog by default does not allow any customization. I'd need to create a full new ListAdapter. It's not worth the difficulty to create it

**Comment by david-allison** [comment]
* https://github.com/ankidroid/Anki-Android/pull/17709
